Update CHANGELOG.md

kuth-chi opened the pull request:
I added tutorial in Khmer language. under 2.7.5 version section.

<!--
Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!

Some guidelines are provided like this, in HTML comments, to expedite the code review before merging your contribution.

First, please make sure that you have read the documentation page on fpdf2 development:
https://pyfpdf.github.io/fpdf2/Development.html

If you're new to contributing to open source projects,
you might find this free video course helpful: http://kcd.im/pull-request
-->

<!-- What changes are being made? (What feature/bug is being fixed here?) -->

e.g. Fixes #0 <!-- This will automatically close issue #0 once the PR is merged: https://help.github.com/en/articles/closing-issues-using-keywords -->

<!-- Have you done all of these things?  -->
**Checklist**:

<!-- To check an item, place an "x" in the box like so: "- [x] Item description"
     Add "N/A" to the end of each line that's irrelevant to your changes -->

- [x] The GitHub pipeline is OK (green), <!-- The maintainers will trigger it if this is your 1st contribution -->
      meaning that both `pylint` (static code analyzer) and `black` (code formatter) are happy with the changes of this PR.

- [x] A unit test is covering the code added / modified by this PR

- [x] This PR is ready to be merged <!-- In your opinion, can this be merged as soon as it's reviewed? Else, this can be turned into a Draft PR -->

- [x] In case of a new feature, docstrings have been added, with also some documentation in the `docs/` folder

- [x] A mention of the change is present in `CHANGELOG.md`

<!-- Feel free to add additional comments, and to ask questions if some of those guidelines are unclear to you! -->

<!--
Once a PR is merged, maintainers will add your name to the contributors table in README.md.
If they forget, or you do not wish this to happen, please mention it.
-->

By submitting this pull request, I confirm that my contribution is made under the terms of the [GNU LGPL 3.0 license](https://github.com/PyFPDF/fpdf2/blob/master/LICENSE).

Lucas-C commented:
Could you please:
1. Add a link to your translation on the line you added
 (the file already contains mentions of previous translations being added, that can serve as examples to you)
2. Integrate this change in the other PR you opened, https://github.com/PyFPDF/fpdf2/pull/799 ?

kuth-chi commented:
> Could you please:
> 
> 1. Add a link to your translation on the line you added
>    (the file already contains mentions of previous translations being added, that can serve as examples to you)
> 2. Integrate this change in the other PR you opened, [Create Tutorial-km-md #799](https://github.com/PyFPDF/fpdf2/pull/799) ?



> I added tutorial in Khmer language. under 2.7.5 version section.
> 
> e.g. Fixes #0
> 
> **Checklist**:
> 
> * [ ]  The GitHub pipeline is OK (green), 
>   meaning that both `pylint` (static code analyzer) and `black` (code formatter) are happy with the changes of this PR.
> * [ ]  A unit test is covering the code added / modified by this PR
> * [ ]  This PR is ready to be merged
> * [ ]  In case of a new feature, docstrings have been added, with also some documentation in the `docs/` folder
> * [ ]  A mention of the change is present in `CHANGELOG.md`
> 
> By submitting this pull request, I confirm that my contribution is made under the terms of the [GNU LGPL 3.0 license](https://github.com/PyFPDF/fpdf2/blob/master/LICENSE).

Hi @Lucas-C 
I have made change the file as structure line of adding new Tutorial-language as following my clone the project into my pc and made it change by create new branch my case "pdf2km" by added and changed as describe but when I tried to merge and push I got the following error as permission issue. How to solve it. 
![image](https://github.com/PyFPDF/fpdf2/assets/69514175/402b8b20-5115-46c4-a421-1a9ec12678d4)

Lucas-C commented:
Hi @kuth-chi

What you want is to push to the branch [patch-3](https://github.com/kuth-chi/fpdf2/tree/patch-3) of your fork ( https://github.com/kuth-chi ), not directly to this repository ( https://github.com/PyFPDF/fpdf2 ) where you don't have the rights to push commits.

The file `.git/config` in your repository should contain something like this:
```
[remote "upstream"]
        url = git@github.com:PyFPDF/fpdf2.git
        fetch = +refs/heads/*:refs/remotes/upstream/*
[remote "origin"]
        url = git@github.com:kuth-chi/fpdf2.git
        fetch = +refs/heads/*:refs/remotes/origin/*
[branch "patch-3"]
        remote = origin
        merge = refs/heads/patch-3
```

The `remote upstream` is optional, but will let you `rebase` easily.
Fore more information, check those documentation pages:
* https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-repository-for-a-fork
* https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
